example.env

```diff
@@ -22,3 +24,52 @@ ETHERSCAN_API_KEY_POLYGON_TESTNET=""
 ETHERSCAN_API_KEY_POLYGON_ZKEVM_GOERLI_TESTNET=""
 ETHERSCAN_API_KEY_POLYGON_ZKEVM=""
 ETHERSCAN_API_KEY_POLYGON=""
+HARDHAT_HTTP_RPC_URL_ARBITRUM_GOERLI_TESTNET=""
```

Comment:
[@api3/chains@3.4.2 is broken](https://github.com/api3dao/chains/pull/80) so I fixed these manually for now